templates/default/atlantis.yaml.erb

```diff
@@ -1,9 +1,5 @@
----
-<% @vars.each do |k,v| %>
-<% next if k == 'config' %>
-<% if v.is_a?(Array) %>
-<%= k %>: <%= v.join(',') %>
-<% next %>
-<% end %>
-<%= k %>: <%= if k == 'log-level' then v.downcase else v end %>
-<% end %>
+# yamllint disable-file
+# ##################### CHEF MANAGED DO NOT EDIT #########################
```

Comment:
If your gonna add it lets make it not a liar for cinc: https://cinc.sh/blog/cooking_with_cinc/

Reply:
Yes very good point i will update it so it doesn't lie for cinc.